src/Microsoft.AspNet.Mvc.Razor/Razor/PreCompileViews/RazorPreCompiler.cs

```diff
@@ -15,39 +18,42 @@ namespace Microsoft.AspNet.Mvc.Razor
 {
     public class RazorPreCompiler
     {
+        private static readonly int MaxDegreesOfParallelism = Environment.ProcessorCount;
```

Comment:
I guess we could once people ask for it? Right now it seems like implementation detail that we don't expose?